### ISSUE TITLE
Look for better quality car boxes

### ISSUE THREAD

**Issue by BGMP** [Re-Volt-America/rva_cars#4]
When compiling all car boxes into the carboxes directory, we only look for two standard names which are almost guaranteed to be present within a cars' directory. A better approach would be trying to fetch a better quality image before defaulting to these two; so, looking for `carbox256.bmp` and probably other low/higher res variants to send to `carboxes/`

https://github.com/Re-Volt-America/rva_cars/blob/8a0dbe891cc273f5d7a7647943bc200b6563bc25/Rakefile#L76-L80

Doing this would mean we don't have to manually arrange them anymore.

ref: https://github.com/Re-Volt-America/Website/pull/38

**Comment by EstebanMz** [comment]
Xarc, as you know, has dozens of his creations included in the pack, he usually let the carbox file with dimensions 128p, or even 85p as default, but also he almost always includes a `carbox256.bmp` file inside the folder. I said almost always because there are some cars of him that include that file inside a subfolder, usually named `additional` _but I have to check it_ thoroughly if the are different folder names. Norfair's works have a similar structure regarding the carboxes.
So in summary, there are at least two new conditions to look for better quality carboxes, first check for `/carbox256.bmp`, then may try for `/additional/carbox256.bmp` and then what's already in the code above, I guess.

**Comment by BGMP** [comment]
> So in summary, there are at least two new conditions to look for better quality carboxes, first check for /carbox256.bmp, then may try for /additional/carbox256.bmp and then what's already in the code above, I guess.

This will likely require a manual re-bundle of the pack. I have removed it now, but originally the clean task used to delete the `additional/` directory present within car folders.

On another note, I added basic support for `carbox256.bmp` retrieval as I mentioned above.

**Comment by BGMP** [comment]
The pack was re-bundled a long time ago. The current finding process for car boxes should be more than enough. In fact, we may not even need this anymore with the new website in place.
